keps/sig-etcd/4331-livez-readyz/kep.yaml

```diff
@@ -4,7 +4,7 @@ authors:
   - "@siyuanfoundation"
   - "@chaochn47"
 owning-sig: sig-etcd
-status: implementable
+status: provisional
```

Comment:
@serathius - FYI

pushing back to provisional as PRR was not filled in and reviewed [to pass improved validation]

Reply:
✅

Reply:
SG